curiouslearning/FeedTheMonsterJS
# fix: CR-174, CR-175 & CR-176

# Changes
- lang/oromo/ftm_oromo.json
- lang/oromo/ftm_somali.json
- lang/oromo/ftm_tigirigna.json

# How to test
- Check all the three languages app title.

Ref: [<JIRA-CODE>](https://curiouslearning.atlassian.net/browse/<JIRA-CODE>)
